javascript/features/gangs/gang.js

```diff
@@ -90,7 +96,20 @@ class Gang {
             return;
 
         setPlayerGangColor(player, useGangColor ? this.color_
-                                                : null);
+            : null);
+    }
+
+    // Returns whether the |player| will use the gang's skin.
+    usesGangSkin(player) {
+        return player.settings
+            .getValue(`${PlayerSetting.CATEGORY.GANG}/${PlayerSetting.GANG.USE_SKIN}`) === true;
```

Comment:
Wondering if this really is better than just typing out the setting name.

```javascript
return player.settings.getValue('gang/use_skin');
```

Looks a lot more readable to me, and as long as the `getValue()` function throws on unknown values, we get the same level of validation.

It further removes the need to include `PlayerSettings` everywhere, removing the layering issue on having to include the file.

Reply:
I think I agree on this. Also due to this not validating whether `.GANG` is defined. `PlayerSetting.CATEGORY` existing doesn't mean `.GANG` exists.